core/designsystem/src/main/java/co/kr/tnt/designsystem/component/button/Button.kt

```diff
@@ -45,14 +47,14 @@ fun TnTTextButton(
             color = type.borderColor(enabled),
         ),
         contentPadding = size.contentPadding,
-        modifier = modifier
-            .height(size.height)
-            // Small, XSmall 버튼을 위해 Button minWidth 설정
-            .defaultMinSize(minWidth = Dp.Hairline),
+        // Small, XSmall 버튼을 위해 Button minWidth, minHeight 설정
+        modifier = modifier.defaultMinSize(minWidth = Dp.Hairline, minHeight = size.height),
```

Comment:
```suggestion
        modifier = modifier.defaultMinSize(
            minWidth = Dp.Hairline, 
            minHeight = size.height
        ),
```

2줄 이상이면 줄바꿈 해주는게 좋을 것 같습니다!